src/api/public/apidocs-new/paths/status_messages.yaml

```
@@ -30,3 +30,31 @@ get:
       $ref: '../components/responses/unauthorized.yaml'
   tags:
     - Status messages
+
+post:
+  summary: Create a status message
+  description: |
+    Create a new status message (Only admins are allowed to add status messages).
+    Possible values for severity: (information, green, yellow, red, announcement).
+    Possible values for scope: (all_users, logged_in_users, admin_users, in_beta_users, in_rollout_users)
+  security:
+    - basic_authentication: []
+  requestBody:
+    description: Attributes for status message
+    content:
+      application/xml; charset=utf-8:
+        schema:
+          $ref: '../components/schemas/status_message.yaml'
+        example:
+          message: Feature number 42 was implemented!
+          severity: announcement
+          scope: all_users
```

```suggestion
  requestBody:
    description: Attributes for status message
    required: true
    content:
      application/xml; charset=utf-8:
        schema:
          $ref: '../components/schemas/status_message.yaml'
        example:
          message: Feature number 42 was implemented!
          severity: announcement
          scope: all_users
```
This looks like the request body should be required.